src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -809,6 +809,24 @@ public function getDocumentsAction(int $orderId)
         ]);
     }
 
+    /**
+     * @AdminSecurity("is_granted(['create', 'update'], request.get('_legacy_controller'))", redirectRoute="admin_orders_index")
```

Comment:
Should be `view` because of `GetOrderForViewing` otherwise, something is confusing

Reply:
I fix the other 2 actions to use `read`
Nice catch